scripts/generate-government-agency-names.py

```
@@ -0,0 +1,276 @@
+            'prefecture':   prefecture,
+            'municipality': municipality,
+            'note':         note,
+        })
+
+    # ── Phase 1: supplementary_valid_names.csv の government_branch を移行 ──
+    print('補助辞書 government_branch の移行中...')
+    with open(SUPP_CSV, encoding='utf-8') as f:
+        supp_rows = list(csv.DictReader(f))
+
+    # note=市区町村: 単純な市区町村名・会計管理者でCNなし → 行政機関ではないのでスキップ
+    # note=都道府県: 都道府県略称（例: 和歌山）→ 厳密辞書の pref_abbr=True で対応
```

Ruff (0.15.4)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)